finagle-serversets/src/test/scala/com/twitter/finagle/serverset2/ZkSessionEndToEndTest.scala

```diff
@@ -27,7 +27,7 @@ class ZkSessionEndToEndTest extends FunSuite with BeforeAndAfter {
   def toSpan(d: Duration): Span = Span(d.inNanoseconds, Nanoseconds)
 
   implicit val patienceConfig =
-    PatienceConfig(timeout = toSpan(1.second), interval = toSpan(zkTimeout))
+    PatienceConfig(timeout = toSpan(5.second), interval = toSpan(zkTimeout))
```

Comment:
Any reasons to increase the timeout?

Reply:
The tests (also existing ones) were flaky for me and increasing timeout helped. I ran them now again a couple of times without any issues so I'm not sure what the failures were about. In any case I reverted all test changes.